_sass/_hca.scss

```diff
@@ -1,4 +1,14 @@
 // TODO: properly incorporate usds web standards for this
+.hca-landing-page {
```

Comment:
This block _shouldn't_ be necessary. Can you try removing it and see if the underline returns?